packages/flutter/lib/src/rendering/custom_layout.dart

```diff
@@ -41,26 +41,42 @@ class MultiChildLayoutParentData extends ContainerBoxParentData<RenderBox> {
 /// Used with [CustomMultiChildLayout], the widget for the
 /// [RenderCustomMultiChildLayoutBox] render object.
 ///
-/// ## Example
+/// Each child must be wrapped in a [LayoutId] widget to assign the id that
```

Comment:
While you're here, should we add a comment on the `id`'s uniqueness requirements here and in CustomMultiChildLayout

Reply:
Done, and in the docs for LayoutId.